### PR TITLE
Modified Japanese translation in 'get-involved' page 

### PR DESCRIPTION
**Pull request by asahina820**
I have modified the Japanese translation page of "get-involved.

- modified join.md to be a natural Japanese translation
- fixed the 'UNスマートマップ' to '国連スマート地図'
- unified the notation of time, local time, and zoom link.

Sorry if I'm wrong:-0

### CODE REVIEW

File: i18n/ja/docusaurus-plugin-content-docs/current/get-involved/join.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-私たちが何をしているのかを見て、[UNスマートマップGitHubの問題](https://github.com/unopengis/7/issues)ページで自己紹介をして貢献を始めることができます。
+### GitHub Issues
+[国連スマート地図のGitHubのIssues](https://github.com/unopengis/7/issues)ページで自己紹介をすることで、私たちの活動を見たり、貢献を始めたりすることができます。
 
-## あなたはどのタイプのUNスマートマッパーですか？
+## あなたはどのタイプの国連スマートマッパーですか？
 
-### スチューデント
-GISやマッピングの経験を積みたい学生ですか？ UNスマートマップのチュートリアルをチェックして、今日からマッピングを始めましょう！
+### 学生
```
Comment (albertkun) [inline i18n/ja/docusaurus-plugin-content-docs/current/get-involved/join.md:18]: @hfu as per @asahina820's revisions for the Japanese version should we use with the original Kanji version of the personas as defined in https://github.com/UNopenGIS/7/issues/158 ?

**Review by albertkun — COMMENTED** [review]
Pending on whether or not we should stick with the the Kanji or the katakana of the SMART persona

**Comment by albertkun** [comment]
One suggestion! Can you modify the Python script that generates the markdown files too? 🙏🙏

I made a new issue for it! 

https://github.com/UNopenGIS/smartmaps/issues/72

File: i18n/ja/docusaurus-plugin-content-docs/current/get-involved/join.md (hunk 1 above)
Comment (hfu) [reply i18n/ja/docusaurus-plugin-content-docs/current/get-involved/join.md:18]: I like the translation by @asahina820 and so we can go ahead!

Comment (asahina820) [reply i18n/ja/docusaurus-plugin-content-docs/current/get-involved/join.md:18]: @albertkun @hfu Thanks for reviewing!
I'm glad my translation was accepted:)

**Pull request merged**